manifest.yml

```diff
@@ -30,7 +30,7 @@ service_definitions:
 parameters: []
 required_env_variables: []
 env_config_mapping:
-  SOLR_SERVER: solr.server
+  SOLR_CP_SERVER: solr.server
```

Comment:
The env var name shouldn't change... The broker only ever needs the control plane endpoint, and it'll be confusing for people configuring the brokerpak if the name of the env var and the name of the config file entry are different! (Plus this change will [break existing staging/prod configuration which uses the old name](https://github.com/GSA/datagov-ssb/blob/7261e274181c02a67f33187bca22d11e5f31ac25/app-setup-solr.sh#L25).)

Reply:
You are right.  I think I changed it not understanding the rework you did to separate KiND from EKS stuff.  I changed it back 👍